test/checkpoint/checkpointer.c

```diff
@@ -482,17 +458,11 @@ verify_consistency(WT_SESSION *session, wt_timestamp_t verify_ts, bool use_check
         }
     }
 
-    /* Pick a reference table: the first table that's not FLCS, if possible; else 0. */
-    reference_table = 0;
-    for (i = 0; i < g.ntables; i++)
-        if (g.cookies[i].type != FIX) {
-            reference_table = i;
-            break;
-        }
-
+    /* Pick a reference table at random. */
+    reference_table = (int)(__wt_random(&g.opts.extra_rnd) % (uint32_t)g.ntables);
```

Comment:
I'm wondering since we’re removing FLCS, the original logic of picking the first non-FLCS table would now simply pick the first table. Is there a reason we’re switching to choosing a random table instead of keeping the first one for consistency with previous behaviour?